src/mandelbrot.rs

```diff
@@ -0,0 +1,104 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by Matt Watson
+// contributed by TeXitoi
+
+use std::io::Write;
+use std::thread;
+const THREADS: usize = 8;
```

Comment:
This probably needs attention as at present size must be an even multiple of THREADS